source/process/release-process.md

```diff
@@ -258,6 +259,7 @@ Once final dot release build is ready to cut:
       - Test the download links before and after updating the page 
     - Update [Mattermost pricing page](https://about.mattermost.com/pricing/) if anything has changed
     - Add the download links to http://docs.mattermost.com/administration/upgrade.html#version-archive  
+    - Update [mattermost-docker](https://github.com/mattermost/mattermost-docker) version 
```

Comment:
We have draft update instructions here: https://github.com/mattermost/mattermost-docker/pull/91/files#diff-aa42d848e7e0ee6100ed717268346bd1R41

Propose opening a separate ticket to update this doc with a link after this is merged and the PR above is merged